frontend/src/components/common/SimpleTable.tsx

```diff
@@ -71,12 +126,16 @@ export default function SimpleTable(props: SimpleTableProps) {
     filterFunction = null,
     emptyMessage = null,
     errorMessage = null,
+    defaultSortingColumn
   } = props;
   const [page, setPage] = React.useState(0);
   const [currentData, setCurrentData] = React.useState(data);
   const rowsPerPageOptions = props.rowsPerPage || [5, 10, 50];
   const [rowsPerPage, setRowsPerPage] = React.useState(rowsPerPageOptions[0]);
   const classes = useTableStyle();
+  const [isIncreasingOrder, setIsIncreasingOrder] = React.useState(!!(defaultSortingColumn &&
+    defaultSortingColumn > 0));
```

Comment:
Does this mean that if we have a defaultSortingColumn than we assume it's in increasing order, otherwise it's not? Why is that (instead of just choosing true or false)?

Reply:
so if we have defaultSortingColumn than we check whether its value is greater than 0 or not and if that's true we know the default sorting order is increasing